_test/test_sqw/test_PixelData_operations.m

```diff
@@ -24,6 +33,9 @@
 
         file_info = dir(obj.test_sqw_file_path);
         obj.page_size = file_info.bytes/6;  % Gives us 6 pages
+
+        obj.pix_in_memory = sqw_test_obj.data.pix;
+        obj.pix_with_pages = PixelData(obj.test_sqw_file_path, obj.page_size);
```

Comment:
this assumes there will always be room for the test data plus 1/6th of the test data in memory: is that true?

Reply:
For the test data it's using, yes. It's only a few Mb. Many of the tests rely on us being able to hold the data in memory (e.g. `test_compute_bin_data_correct_output_in_memory_mex_1_thread`), so using data that doesn't fit in memory does not make sense for this test suite.